src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -132,27 +141,36 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 					this.logService.error('Unexpected query event type "' + event.type + '" sent');
 					break;
 			}
-			self._cd.detectChanges();
+			//self._cd.detectChanges();
 		});
 
-		this.dataService.onAngularLoaded();
+		this.dataService.onComponentLoaded();
+	}
+
+
+	public render(container: HTMLElement): void {
+		this.nativeElement.style.width = '100%';
+		this.nativeElement.style.height = '100%';
+
+		container.appendChild(this.nativeElement);
 	}
 
+
 	protected initShortcuts(shortcuts: { [name: string]: Function }): void {
 		// TODO add any Edit Data-specific shortcuts here
 	}
 
-	public ngOnDestroy(): void {
+	public onDestroy(): void {
 		this.baseDestroy();
 	}
 
-	handleStart(self: EditDataComponent, event: any): void {
+	handleStart(self: EditDataGridPanel, event: any): void {
 		self.dataSet = undefined;
 		self.placeHolderDataSets = [];
 		self.renderedDataSets = self.placeHolderDataSets;
-		this._cd.detectChanges();
+		//this._cd.detectChanges();
```

Comment:
dead code.  please remove all commented out code before sending for final review.